permission/permissions.go

```diff
@@ -1,53 +0,0 @@
-package permission
```

Comment:
Is this just a leftover?

Reply:
Yes that is an old one, some duplicates will come of this as the different versions happen, I should merge from pull 10 in a but